tests/trustyai/conftest.py

```diff
@@ -0,0 +1,111 @@
+from typing import Generator, Any
+
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from ocp_resources.resource import get_client
+from ocp_resources.secret import Secret
+from ocp_resources.service import Service
+
+from tests.trustyai.utils.minio import MinioPod, MinioService, MinioSecret
+
+
+MINIO: str = "minio"
+OPENDATAHUB_IO: str = "opendatahub.io"
+
+
+@pytest.fixture(scope="session")
+def admin_client() -> DynamicClient:
+    return get_client()
+
+
+@pytest.fixture(scope="class")
+def model_namespace(request: Any, admin_client: DynamicClient) -> Namespace:
+    with Namespace(
+        client=admin_client,
+        name=request.param["name"],
+    ) as ns:
+        ns.wait_for_status(status=Namespace.Status.ACTIVE, timeout=120)
+        yield ns
+
+
+@pytest.fixture(scope="class")
+def minio_pod(admin_client: DynamicClient, model_namespace: Namespace) -> Generator[MinioPod]:
```

Comment:
```suggestion
def minio_pod(admin_client: DynamicClient, model_namespace: Namespace) -> Pod:
```
i would go with this approach; wdyt?